Add demo docs and resources (#2)
seqeralabs/showcase: README.md

# `seqeralabs/showcase` Workspace on Seqera Platform

## Infrastructure as code

> Infrastructure as Code (IaC) provides the ability to provision and manage infrastructure through configuration files. Having IaC enables automation in set up, cconsistency and standardization in infrastructure, and documentation of how resources are configured.

Reviewer: ```suggestion
Infrastructure as Code (IaC) provides the ability to provision and manage infrastructure through configuration files. Having IaC enables automation in set up, consistency and standardization in infrastructure, and documentation of how resources are configured.
```

This guide describes how to create resources on Seqera Platform through IaC using two methods: (1) Seqera Platform CLI, and (2) seqerakit.

### 1. Using Seqera Platform CLI

> Most [Seqera Platform](https://seqera.io/platform/) entities such as Pipelines, Compute Environments and Datasets can be exported in JSON format via the [Seqera Platform CLI](https://github.com/seqeralabs/tower-cli#nextflow-tower-cli). This is very useful for creating infrastructure as code to store the exact configuration options used to create these entities, and to share and track changes over time.

Reviewer: ```suggestion
Most [Seqera Platform](https://seqera.io/platform/) entities such as Pipelines, Compute Environments and Datasets can be exported in JSON format via the [Seqera Platform CLI](https://github.com/seqeralabs/tower-cli#nextflow-tower-cli). This is very useful for creating IaC to store the exact configuration options used to create these entities, and to share and track changes over time.
```

- Credentials for AWS, Azure, and Google have already been set-up in Seqera Platform

### 2. Using `seqerakit`

The [Seqera Platform](https://https://seqera.io/platform/) entities represented above in JSON can also be represented as YAML configuration files. These configuration YAMLs can be used by `seqerakit`, a Python package wrapping the Seqera Platform CLI, to define and automate how entities should be created.

This repository also contains YAML configuration files defining creation of Seqera Platform entities on the `seqeralabs/showcase` Workspace with `seqerakit`:

- [Compute Environments](seqerakit/compute-envs/)
- [Pipelines](seqerakit/pipelines/)
- [Datasets](seqerakit/datasets/)

#### Prerequisites

To use `seqerakit`, you will need to have:

- The [Seqera Platform CLI](https://github.com/seqeralabs/tower-cli#1-installation) installed and `tw info` has been configured as described above
- `TOWER_API_ENDPOINT` and `TOWER_ACCESS_TOKEN` environment variables set
  Credentials for AWS, Azure, and Google have already been set-up in Seqera Platform
- The package installed via:

```bash
conda install seqerakit
```

Or:

```bash
pip install seqerakit
```

> Please refer to the package's [installation guide](docs/installation.md) for more details on the required dependencies.

Reviewer: The link to the installation guide is broken.

#### Running `seqerakit`

> Similar to executing the scripts above to import and create Pipelines, Compute Environments, and Datasets that are defined in JSON onto Seqera Platform, the [YAML files](#using-seqerakit) provided in this repository can also be used create `seqeralabs/showcase` entities on the Platform. For example:

Reviewer: ```suggestion
Similar to executing the scripts above to import and create Pipelines, Compute Environments, and Datasets that are defined in JSON onto Seqera Platform, the [YAML files](#using-seqerakit) provided in this repository can also be used to create `seqeralabs/showcase` entities on the Platform. For example:
```
